release_notes/Service_Fabric_ReleaseNotes_90CU9.md

```diff
@@ -0,0 +1,61 @@
+# Microsoft Azure Service Fabric 9.0 Cumulative Update 9.0 Release Notes
+
+This release will only be available through manual upgrades. Clusters set to automatic upgrades will not receive this release. For how to configure upgrades, please see [classic](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-upgrade) or [managed](https://docs.microsoft.com/en-us/azure/service-fabric/how-to-managed-cluster-configuration) documentation.
+
+## Contents
+* [Service Fabric Packages and Versions](#service-fabric-packages-and-versions)
+* [Service Fabric Feature and Bug Fixes](#service-fabric-feature-and-bug-fixes)
+* [Retirement and Deprecation Path Callouts](#retirement-and-deprecation-path-callouts)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+## Service Fabric Packages and Versions
+The following packages and versions are part of this release:
+
+| Service | Platform | Version |
+|---------|----------|---------|
+|Service Fabric Runtime| Ubuntu 18 <br> Ubuntu 20 <br> Windows | 9.0.1463.1 <br> 9.0.1463.1 <br> 9.0.1526.9590 |
+|Service Fabric for Windows Server|Service Fabric Standalone Installer Package | 9.0.1526.9590 |
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 6.0.1526  <br> 9.0.1526 <br> 6.0.1526 <br> 6.0.1526 |
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.1 |
+
+## Service Fabric Feature and Bug Fixes
+| Versions | IssueType | Description | Resolution | 
+|-|-|-|-|
+| **Windows - 9.0.1526.9590<br>Ubuntu 18 - 9.0.1463.1<br>Ubuntu 20 - 9.0.1463.1** | **Feature** | Key Value Store (KVS) | **Brief Description**:  KVS backed partitions, including Actors and System Services, experience quorum loss due to StoreLVIDLimitHit error code (0x80071d4d), indicating the depletion of values for a monotonically increasing ID field. This results in unavailability of the partition until a manual offline maintenance task is executed to resolve the issue. <br>**Solution**: To address the error, KVS replicas now restart automatically and initiate the maintenance task (Extensible Storage Engine database compaction) during the next open. While there may be temporary availability loss in partitions depending on the task duration, no manual intervention is required to mitigate the issue.
```

Comment:
While there may be temporary availability loss in partitions depending on the task duration, no manual intervention is required to mitigate the issue.

This feels very open ended, can we quantify with any particular typical experience that is expected from testing this?